### PR TITLE
adds a spinner to indicate that a profile is fetching

### LINKED ISSUE #1485: Indicators for profile updates (spinner) and query invalid for column insights in the inspector while modeling
subjective experience of waiting for profiles has degraded. Spinners for each column profile would help users understand that the profile being shown is stale and they should wait for the new data.  I should also have some sense of whether I am not waiting for data and the query is invalid.

### PR DESCRIPTION
**Pull request by hamilton**
closes #1485 

Kapture 2022-12-21 at 14 30 08

### CODE REVIEW

**Review by magorlick — APPROVED** [review]
lgtm

File: web-local/src/lib/components/column-profile/column-types/NumericProfile.svelte [hunk 1]
```diff
@@ -41,13 +44,18 @@
...
+  $: rug = useRuntimeServiceGetRugHistogram(
     $runtimeStore?.instanceId,
     objectName,
-    columnName,
-    active
+    { columnName, priority: getPriorityForColumn("rug-histogram", active) },
+    {
+      query: {
+        select($query) {
+          return $query?.numericSummary?.numericOutliers?.outliers;
```
Comment (ericpgreen2) [inline web-local/src/lib/components/column-profile/column-types/NumericProfile.svelte:54]: I think convention is for select functions to be lambda functions, as in this example: https://tkdodo.eu/blog/react-query-data-transformations#3-using-the-select-option

File: web-local/src/lib/components/column-profile/column-types/NumericProfile.svelte [hunk 2]
```diff
@@ -83,9 +94,11 @@
     copyToClipboard(columnName, `copied ${columnName} to clipboard`)}
   {type}
 >
-  <DataTypeIcon slot="icon" {type} />
+  <ColumnProfileIcon slot="icon" isFetching={fetchingSummaries} {type} />
+
+  <!-- <DataTypeIcon slot="icon" {type} /> -->
```
Comment (ericpgreen2) [inline web-local/src/lib/components/column-profile/column-types/NumericProfile.svelte:99]: Leftover comment

File: web-local/src/lib/components/column-profile/queries.ts [hunk 3]
```diff
@@ -165,28 +172,16 @@ export function getNumericHistogram(
...
-  const outliersQuery = useRuntimeServiceGetRugHistogram(
+  return useRuntimeServiceGetNumericHistogram(
     instanceId,
     objectName,
-    { columnName, priority: getPriorityForColumn("rug-histogram", active) }
+    { columnName, priority: getPriorityForColumn("numeric-histogram", active) },
+    {
+      query: {
+        select(query) {
+          return query?.numericSummary?.numericHistogramBins?.bins;
```
Comment (ericpgreen2) [inline web-local/src/lib/components/column-profile/queries.ts:182]: Consider a lambda function as mentioned above

**Review by ericpgreen2 — APPROVED** [review]
Left a few nits, otherwise looks good 👍

**Pull request merged**